src/cloudwatch-appsignals-mcp-server/awslabs/cloudwatch_appsignals_mcp_server/enablement_tools.py

```diff
@@ -0,0 +1,142 @@
+        app_directory: ABSOLUTE path to the application code directory (e.g., /home/user/project/app)
+
+    Returns:
+        Enablement guide with step-by-step instructions
+    """
+    logger.debug(
+        f'get_enablement_guide called: platform={platform}, language={language}, '
+        f'iac_directory={iac_directory}, app_directory={app_directory}'
+    )
+
+    platform = platform.lower().strip()
+    language = language.lower().strip()
+
+    valid_platforms = ['ec2', 'ecs', 'lambda', 'eks']
+    if platform not in valid_platforms:
+        return f"Error: Invalid platform: '{platform}'. Valid platforms are: {', '.join(valid_platforms)}"
+
+    valid_languages = ['python', 'nodejs', 'java', 'dotnet']
+    if language not in valid_languages:
+        return f"Error: Invalid language '{language}'. Valid languages are: {', '.join(valid_languages)}"
+
+    # Validate that paths are absolute
+    iac_path = Path(iac_directory)
+    app_path = Path(app_directory)
+
+    if not iac_path.is_absolute():
+        error_msg = (
+            f'Error: iac_directory must be an absolute path.\n\n'
+            f'Received: {iac_directory}\n'
+            f'Please provide an absolute path (e.g., /home/user/project/infrastructure)'
+        )
+        logger.error(error_msg)
+        return error_msg
+
+    if not app_path.is_absolute():
+        error_msg = (
+            f'Error: app_directory must be an absolute path.\n\n'
+            f'Received: {app_directory}\n'
+            f'Please provide an absolute path (e.g., /home/user/project/app)'
+        )
+        logger.error(error_msg)
+        return error_msg
+
+    logger.debug(f'IaC path: {iac_path}')
+    logger.debug(f'App path: {app_path}')
+
+    if not iac_path.exists():
+        error_msg = (
+            f'Error: IaC directory does not exist: {iac_path}\n\n'
+            f'Please provide a valid path to your infrastructure code directory.'
+        )
+        logger.error(error_msg)
+        return error_msg
+
+    if not app_path.exists():
+        error_msg = (
+            f'Error: Application directory does not exist: {app_path}\n\n'
+            f'Please provide a valid path to your application code directory.'
+        )
+        logger.error(error_msg)
+        return error_msg
+
+    guides_dir = Path(__file__).parent / 'enablement_guides'
+    template_file = guides_dir / 'templates' / platform / f'{platform}-{language}-enablement.md'
+
+    logger.debug(f'Looking for enablement guide: {template_file}')
+
+    if not template_file.exists():
+        logger.warning(f'Enablement guide not found: {template_file}')
+        return (
+            f"Error: Enablement guide for platform '{platform}' and language '{language}' "
+            f' is not yet available.\n\n'
+            f'Currently supported combinations:\n'
+            f'- EC2 + Python\n\n'
+            f'Requested: {platform} + {language}'
```

Comment:
So what should agent do? Lets not hardcode EC2/Python.
```suggestion
            f"Error: Enablement guide for platform '{platform}' and language '{language}' "
            f' is not yet available.\n\n'
            f'Requested: {platform} + {language}'
```

Reply:
Made this change:

```
    if not template_file.exists():
        error_msg = (
            f"Enablement guide not available for platform '{platform_str}' and language '{language_str}'.\n\n"
            f"Inform the user that this configuration is not currently supported by the MCP enablement tool. "
            f"Direct them to AWS documentation for manual setup:\n"
            f"https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Application-Signals-Enable.html"
        )
        logger.error(error_msg)
        return error_msg
```

Agent should now inform customer this platform/language is not supported by our enablement tool yet and direct them to our public docs for manual instructions.